compiler/src/dotty/tools/dotc/typer/Inliner.scala

```diff
@@ -104,14 +94,14 @@ object Inliner {
    *                     to have the inlined method as owner.
    */
   def registerInlineInfo(
-      sym: SymDenotation, treeExpr: Context => Tree)(implicit ctx: Context): Unit = {
-    sym.unforcedAnnotation(defn.BodyAnnot) match {
+      inlined: Symbol, treeExpr: Context => Tree)(implicit ctx: Context): Unit = {
```

Comment:
Docs should be updated. Maybe call it `inlineSym`, since it's not inline**d** at this point?